lib/rubocop/config_loader.rb

```diff
@@ -58,7 +58,7 @@ def merge(base_hash, derived_hash)
 
       def base_configs(path, inherit_from)
         configs = Array(inherit_from).map do |f|
-          f = File.join(File.dirname(path), f) unless f.start_with?('/')
+          f = File.expand_path(f, File.dirname(path)) unless f.start_with?('/')
```

Comment:
Ok this is fixed and I squashed commits and rebased with master so I think everything should be good to go!